package.json

```diff
@@ -17,7 +17,7 @@
     "lint:css": "sass-lint 'app/**/*.scss' -v -q",
     "nuke": "npm run clean && rm -rf node_modules/",
     "start:dev": "concurrently \"npm run start:redis\" \"cross-env LOGGER_LEVEL=info babel-node ./app/server\"",
-    "start:redis": "./startRedis.sh",
+    "start:redis": "bash startRedis.sh",
```

Comment:
just out of interest what's the benefit in using `bash` here instead of `./`?

Reply:
That Windows users can run this 😄

Reply:
I assume it will not be problematic on Macs/Linux boxes, but, just to be 100% sure, would you please try it out?

Reply:
oh if windows users can use it then do we need the powershell version then?

Reply:
Only if the win user doesn't have bash - and as far as I'm aware all our windows FE users  (there's so many I know) have bash installed. Therefore we should be able to 'get away' without creating windows versions of start-up scripts in package.json.

Reply:
ok thanks for clearing that up Adrian. Do you mind if you remove the powershell version from this PR and just keeping the shell changes and see if that works when you run the app on your windows machine?

Just need to check that the redis docker container was created and started please

Reply:
Sounds good 👍 

Just gave it a try and works as expected.

`npm run start:dev` spins up the redis container.

Reply:
awesome dude. Thanks for doing that and apologies about making you create the powershell script when it wasn't needed 😂

Reply:
😂 It's alright, its not like I wasted a day on making the script. Glad we didn't do anything that wasn't needed though.